### ISSUE TITLE
Missing a minus sign for negative values

### ISSUE THREAD

**Issue by exdeniz** [ThisIsManta/vscode-stylus-supremacy#5]
example
```
.filterItem
    height 40px
    margin-right 6px
    margin-bottom 5px
    padding 0 20px
    border 1px solid #f0f0f2
    color #000
    font-family 'graphik' Arial Helvetica sans-serif
    font-size 14px
    font-weight 300
    line-height 41px
    text-transform uppercase
    cursor pointer
    transition all .4s
    &Text
        box horizontal middle
        position relative
        span
            margin-right 18px
        &:after
            display block
            position absolute
            bottom -4px
            left -21px
            opacity 0
            visibility hidden
            background-color #f0f0f2
            content ''
            transition all .4s
            size calc(100% + 42px) 6px
            transition-delay .1s
```
on format
```
.filterItem
    height 40px
    margin-right 6px
    margin-bottom 5px
    padding 0 20px
    border 1px solid #f0f0f2
    color #000
    font-family 'graphik' Arial Helvetica sans-serif
    font-size 14px
    font-weight 300
    line-height 41px
    text-transform uppercase
    cursor pointer
    transition all .4s
    &Text
        box horizontal middle
        position relative
        span
            margin-right 18px
        &:after
            display block
            position absolute
            bottom 4px
            left 21px
            opacity 0
            visibility hidden
            background-color #f0f0f2
            content ''
            transition all .4s
            size calc(100% + 42px) 6px
            transition-delay .1s
```
see &:after bottom and left properties

**Comment by ThisIsManta** [comment]
It must have something to do with `insertLeadingZeroBeforeFraction`. I will fix this ASAP. Please bear with me.

**Comment by ThisIsManta** [comment]
This issue has been fixed at **version 0.2.3**. Please update the extension.

Cheers!

**Comment by exdeniz** [comment]
Thanks, it's work fine!
